### ISSUE TITLE
Improve Mochawesome report for Cypress accessibility tests

### ISSUE THREAD

**Issue by ddzz** [department-of-veterans-affairs/va.gov-team#29913]
# Description
Currently the Mochawesome report generated for the daily Cypress accessibility scan only lists whether or not there was an accessibility issue found on a page. In order to view the specific failure(s), the raw test output on GitHub Actions needs to be searched.

This task is to improve the format of the Mochawesome report so that the report shows the specific failure(s) found so that the GitHub Actions logs do not need to be searched.

# Acceptance Criteria
- [ ] Stakeholders can view the details of a Cypress accessibility scan failure in the Mochawesome report generated for the scan

**Comment by ddzz** [comment]
https://github.com/department-of-veterans-affairs/content-build/pull/616

**Comment by pjhill** [comment]
The [new report](https://testing-tools-testing-reports.s3-us-gov-west-1.amazonaws.com/a11y-reports/7cb50f51-7eb1-422e-8eb7-a37386200fdb.html) looks great!
